app/docs/0.11.x/clustering.md

```diff
@@ -4,54 +4,317 @@ title: Clustering Reference
 
 # Clustering Reference
 
-Multiple Kong nodes using the same datastore belong to the same "Kong cluster".
-
 A Kong cluster allows you to scale the system horizontally by adding more
-machines to handle a bigger load of incoming requests, and they all share the
-same data since they point to the same datastore.
+machines to handle a bigger load of incoming requests. They will all share the
+same configuration since they point to the same database.
+
+Kong nodes pointing to the **same datastore** will be part of the same Kong
+cluster.
+
+To efficiently use your Kong cluster, you need a **load-balancer** in front of
+your Kong cluster, to efficiently distribute your traffic across your
+available nodes.
```

Comment:
"To efficiently use your Kong cluster, you need a **load-balancer** in front of
your Kong cluster, to efficiently distribute your traffic across your
available nodes." 
->
"You need a **load-balancer** in front of
your Kong cluster to distribute your traffic across your
available nodes."